lib/components/Carleton360/styles.css

```diff
@@ -0,0 +1,18 @@
+.cu-carleton360 .cu-pageheader > *:last-child {
+  @apply mb-4;
+}
+
+.cu-carleton360 .cu-pageheader > h1:last-child,
+.cu-carleton360 .cu-pageheader > h2:last-child {
+  @apply mb-2;
+}
+
+.cu-carleton360-dark,
+.cu-carleton360-image,
+.cu-carleton360-wave {
+  @apply text-white;
+}
```

Comment:
When duplicating code makes sure you review to clean up stuff that is not being used to keep code lean